eventmesh-dashboard-core/pom.xml

```diff
@@ -40,6 +40,12 @@
     </properties>
 
     <dependencies>
+        <!-- Spring Boot Starter -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
```

Comment:
How about leaving `spring-boot-starter-web` at `common` module for now? Adding the
spring-boot-starter-web artifacts to `core` module seems too deliberate.